src/mapnode.h

```diff
@@ -293,6 +293,11 @@ struct alignas(u32) MapNode
 	void getCollisionBoxes(const NodeDefManager *nodemgr,
 		std::vector<aabb3f> *boxes, u8 neighbors = 0) const;
 
+	/*
+		Return true if Have big selection boxes
+	*/
+	bool haveBigSelectionBox(const NodeDefManager *nodemgr) const;
```

Comment:
```suggestion
	/// Returns `true` when an extent is > BS/2.
	bool haveBigSelectionBox(const NodeDefManager *nodemgr) const;
```

Also: this function might be superfluous. `Environment` already has access to NodeDefManager and MapNode to retrieve this value when needed.

Reply:
> Also: this function might be superfluous.

It is. Good call, removed it.